FloofStation V31.1

M3739 opened the pull request:
<!--
This is a semi-strict format, you can add/remove sections as needed but the order/format should be kept the same
Remove these comments before submitting
-->

# Description

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

### Floof Station, Version 31.1 (Yes, we're doing this.)

_This is the 32nd release for Floof Station. Consisting largely of QOL and small changes. This would be the first deployment where it features a minor component of the version number._

Included PRs:
- #1273 
- #1281 
- #1268 
- #1286 
- #1296 
- #1274 
- #1288
- #1287 
- #1293 
- #1291
- #1298
- #1289 
- #1284 
- #1280 
- #1255 
- #1208
- #1279
- #1285 
- #1300 
- #1306 
- #1221
- #1303 
- #1294
- #1307

Cprn-Sprkl committed:
Add interface (#1273)

Adds the instrument interface to revenants so they can use their sing ability.

Commit:
Automatic Changelog Update (#1273)

Mnemotechnician committed:
Fixes and Tweaks From Vulpstation (#1281)

* Motherfucking lowercase (#41)

(cherry picked from commit e21ab42e870084ff379454a2c7d59b7c983119bd)

* Fix SheetRGlass Recipe (#14)

* Fix SheetRGlass recipe

* Update sheet.yml

(cherry picked from commit 56cda213f71158a6213c926bf916217725c8409c)

* Update changelog.yml: don't continue on error

(cherry picked from commit 71b62827b6987977b4cf76f4dffe00ee6e7fa6aa)

* don't test the suicide system, it's broken due to shitmed

(cherry picked from commit df9dd17a35104fed834d197023fa56279feacf01)

* Don't require maps to have spawners for all jobs

Commit:
Automatic Changelog Update (#1281)

JFerrix committed:
Fixes things (#1268)

I fucking hate github, I wrote a summary but git just didnt give a shit

Commit:
Automatic Changelog Update (#1268)

Mnemotechnician committed:
Constructible Holopads and Lathes (#28) (#1286)

* Constructible holopads and lathes

* Slight tweak

* I thought I had pushed those changes.

* fuck containers.

Commit:
Automatic Changelog Update (#1286)

ItsDzordz committed:
4 New Plushies (#1296)

* 3 New Plushies

This pr adds 3 new plushies, as well as adding them to the random plsuhie pool and prize vendor

* Swaping sounds on Hardcore DJ plushie

swapped the puching sound with the on use sound for Hardcore DJ plushie

* Name change

Slime Bartender plushie > Slimetender plushie

* Adds Hos Plushie + qm plushie sound swap-

added the hos plushie
swapped sound on the qm plushie from human scream to kaching

* test re run

Commit:
Automatic Changelog Update (#1296)

Cprn-Sprkl committed:
Mimeguin Fix: Surgery (#1274)

Added surgery interface to mimeguins.

Commit:
Automatic Changelog Update (#1274)

Mnemotechnician committed:
Also make the shark squeak when attacking (#1288)

Mnemotechnician committed:
More Windows (#31) (#1287)

* add missing construction comps

add missing construction comps to n14 fancy wood window variants

* new buildable windows

new buildable windows

* attempt fix

attempted fix at build fail/yaml linter fail

* uploaded wrong file like a dumb dumb

uploaded wrong file like a dumb dumb

* fix glass

changed glass1 to sheetglass1

* wood windows now require woodplanks as well as glass

---------

Co-authored-by: Dunrab <72047814+Dunrab@users.noreply.github.com>

Commit:
Automatic Changelog Update (#1287)

Mnemotechnician committed:
Fix StandingState mispredicts (#1293)

Commit:
Automatic Changelog Update (#1293)

Mnemotechnician committed:
Update ChatSystem.cs (#1291)

Commit:
Automatic Changelog Update (#1291)

ashkitten committed:
Fix Sign Language Bugs (#1298)

* revert !RequireSpeech ignoring CanSpeak

this caused sign language to be usable while asleep. will need further rework to be reintroduced.

* fix sign language whispers

use InRangeUnOccluded instead of InRangeUnobstructed, and change an errant return to a continue

KyuPolaris committed:
invasion of the rodents (#1289)

KyuPolaris committed:
fixed pride items (#1284)

Cprn-Sprkl committed:
Tajaran Fix: Custom Species & Missing Markings Slots (#1280)

* Tajaran Fix: Custom Species

Adds the one line needed to allow for custom species while selecting Tajaran.

* Upgrade Tajaran Markings

Allows Tajaran to correctly use face, underwear, undershirt, and wing markings. Also adds the TailBehind and TailOversuit marking layers for use with the tail split system.

Raikyr0 committed:
Free the Fridge (#1255)

* Updated the Fridge

* Update lockers.yml

* Update lockers.yml

* Update lockers.yml

ashkitten committed:
Use Encryption Keys Instead of Intrinsic Radio Channels for pAI (#1208)

* use encryption keys instead of intrinsic radio channels for pAI

* fix test failures

Raikyr0 committed:
Antag Advantage Removal (#1301)

* Update lawyer.yml

* Update paramedic.yml

* Update quartermaster.yml

* Update service_worker.yml

* Update salvage_specialist.yml

* Update research_director.yml

* Update station_engineer.yml

* Update cargo_technician.yml

* Update head_of_personnel.yml

* Update senior_engineer.yml

* Update technical_assistant.yml

* Update atmospheric_technician.yml

* Update chief_medical_officer.yml

* Update inventory_specialist.yml

* Update clerk.yml

* Update assistant.yml

* Update assistant.yml

M3739 commented:
<img width="486" height="382" alt="image" src="https://github.com/user-attachments/assets/2d5aab32-2398-4d03-8e34-5e05e8ee95c2" />

ashkitten commented:
can we please also pull in #1279 and #1303? these have been very relevant to gameplay this week

ashkitten committed:
update max allowed basic languages to 2 (#1279)

Commit:
Automatic Changelog Update (#1298)

Commit:
Automatic Changelog Update (#1289)

Commit:
Automatic Changelog Update (#1284)

Commit:
Automatic Changelog Update (#1280)

Commit:
Automatic Changelog Update (#1255)

Commit:
Automatic Changelog Update (#1208)

Commit:
Automatic Changelog Update (#1301)

Commit:
Automatic Changelog Update (#1279)

Mnemotechnician committed:
Port the Tilecutter (#44) (#1285)

* Cutter machine & steel mozaic item tiles (#30431)

* no texture machine

* done

* delete wip

* bug fixes

* oops

* CutterMachineCircuitboard
Glass: 500

* Fixes

---------

Co-authored-by: Ko4ergaPunk <62609550+Ko4ergaPunk@users.noreply.github.com>

Commit:
Automatic Changelog Update (#1285)

M3739 committed:
Task Acumen II (Admin Updates No. 1) (#1300)

* Allow for respawn and erase verb to be used on offline players. Also minor  rewrite on respawn command and new erase command (#30433)

* Localize respawn command, allow for it to use userids, and make it use [Dependency] attributes

* Make respawn verb available for offline players

* Make erase available for offline players

A thousand admins rejoice

* Reorder verbs in code

* Add erase command

* Fix localisation for erase command

* Address reviews and add completion to respawn command

* Complete reviews which I forgor

(cherry picked from commit 9a63144636f9ef272d955f1e131b17233c6ec5da)

* Floofication of Wizden 30433

* Sanitize admin UI input for role bans (#35422)

if job does not exist, log it

(cherry picked from commit 26a0bfbbc5b641d431705fa536a612958aa8c9bf)

* Restrict `MobObserver` from having global hearing.

* Better admin note popups text visibility (#29909)

* Better admin note popup text visibility

* Bring buttons closer to the border

(cherry picked from commit bb673649000315ddaa226342a795ac65e1f65367)

* how did that get there?

* Update comments.

* Modify `VaporizeCheater` to also get the NetUserID for the `Erase` method.

---------

Co-authored-by: nikthechampiongr <32041239+nikthechampiongr@users.noreply.github.com>
Co-authored-by: Killerqu00 <47712032+Killerqu00@users.noreply.github.com>
Co-authored-by: Winkarst-cpu <74284083+Winkarst-cpu@users.noreply.github.com>

Commit:
Automatic Changelog Update (#1300)

M3739 committed:
Fix Changelogs Because We Forgor (#1306)

* Quick fix ig

* Comment out Mnemo's admin CL

* Update Floof.yml

ashkitten committed:
Show Carried Entities in Examine Text, Translate Numbers To Words (#1221)

* show cuffed hands, carried and dragged entities in examine

* sanitize numbers as words in chat

Commit:
Automatic Changelog Update (#1221)

Stealthii committed:
Add Character Traits/Loadout to Paradox Clones (#1303)

* Fix for Paradox Anomaly traits

* Add original character loadout to clone

* Add Floof landmark comments

Co-authored-by: Mnemotechnican <69920617+Mnemotechnician@users.noreply.github.com>

---------

Co-authored-by: Mnemotechnican <69920617+Mnemotechnician@users.noreply.github.com>

Commit:
Automatic Changelog Update (#1303)

Memeji reviewed (approved):
Tested locally, everything ran and worked.
Tried to test what I could and it looks all good and ready!

M3739 committed:
Task Uplift I (Upstream Merge, No. 1) (#1294)

* Martial Artist Rework + Fix MeleeWeapon Wizmerge Bugs (#1560)

Reworks the Martial Artist trait, making it a more impactful and
visually distinct trait.
- Left-clicks are now single-target power attacks, requiring less aim
than before.
- **50%** damage bonus reduced to **20%** damage bonus (same overall DPS
with next change).
- Gain **25%** attack rate bonus.
- The attack rate bonus helps make the trait feel and look distinct from
non-Martial Artist melee attacks.
- **50%** range bonus reduced to **10%** range bonus.
- The damage bonus is now also applied to Asphyxiation and Poison
damage, which Lamia unarmed attacks deal.
- Trait cost increased from **-3** points to **-5** points.
- Striking Calluses (which requires Martial Artist) cost reduced from
**-4** points to **-3** points to prevent the Martial Artist/Striking
Calluses combo from being too expensive. The combo used to cost **-7**
points, now it is **-8** points.

The reworked Martial Artist trait is also given to the Boxers, Martial
Artists and Gladiators.

Also reverted some wizmerge messery that messed up the melee attack rate
**again**, and messed up pistol whipping by making the cooldowns of
gunshots and melee attacks intertwined.

Also while we're at it, Natural Weapons Removal has been disabled for
all species whose damage is pure Blunt, including Diona, Dwarf, Arachne
and IPC. (IPC have 6 blunt so the trait would literally be a 0-point
negative trait for them)

A new trait function has been added for Martial Artist:
`TraitModifyUnarmed`, which modifies the player entity's
`MeleeWeaponComponent`.

The Claws, Talons, Natural Weapon Removal, and Striking Calluses traits
have also been refactored under the hood to use `TraitModifyUnarmed`,
instead of replacing `MeleeWeaponComponent` which would wipe out all the
changes made by the Martial Artist trait.

![martialartist](https://github.com/user-attachments/assets/de51530f-5f2a-41a8-b686-c12f746b5e4a)

https://github.com/user-attachments/assets/7890aac2-2c74-4abd-be9f-b28c2922c865

![strikingcalluses](https://github.com/user-attachments/assets/bc15425e-3460-49f2-88f5-840c686e0053)

![naturalweaponsremoval](https://github.com/user-attachments/assets/583adaa4-48c0-46e3-882b-1bb0f470018c)

:cl: Skubman
- add: Martial Artist Rework: Martial Artist now costs 5 points, but it
turns all unarmed melee attacks into single-target power attacks, with
20% bonus damage, 25% bonus attack rate and 10% bonus attack range.
- tweak: The reworked Martial Artist trait is now given for free to
Boxers, Martial Artists, and Gladiators.
- tweak: Martial Artists (the job) and Gladiators can now select the
Striking Calluses trait.
- tweak: The Martial Artist trait now applies bonus damage to the
Lamiae's unarmed Asphyxiation and Poison damage.
- tweak: The cost of Striking Calluses has been reduced from 4 points to
3 points.
- fix: Fixed a bug where slow weapons were fast and fast weapons were
slow.
- fix: You can pistol whip (right-click melee) immediately after firing
a gun again, and the cooldown on firing the gun after pistol whipping is
always 0.528 seconds again.
- fix: Prevented Dionas, Arachnae and IPCs, who all have pure Blunt
damage from selecting the redundant Natural Weapons Removal trait.

(cherry picked from commit 6c43d005e0804fe29007371a46a56e27bccbd4f8)

* Floofication of 1560

* Split All CVars Into Separate Files (#1587)

This PR splits all the CVars into separate files. Unfortunately I need
to hella clean this up by restoring all the CVars that are different on
EE, because cherrypicking it reset everything to Wizden's Defaults
(Including CVars that no longer exist on EE, and removing CVars from
systems I codeown now, such as Atmos, Guns, and Melee).

No changelog because this isn't player facing. :)

---------

Co-authored-by: Simon <63975668+Simyon264@users.noreply.github.com>
(cherry picked from commit 357f8378c09ccb9f172fc720fd61deb25bc168ea)

* Floofication of 1587

* Better Progress Bar (#1648)

# Description

DoAfter progress bar like in ss13 forks.

---

![2025-01-24_01-04_2](https://github.com/user-attachments/assets/df524426-cecd-4a32-a3b3-f588f11684b1)

![2025-01-24_01-03](https://github.com/user-attachments/assets/62dcb13a-4259-4980-8aa2-87de6e77a226)

![2025-01-24_01-04](https://github.com/user-attachments/assets/493c0c54-4225-40c9-8518-4da613bb2e12)

---

# Changelog

Fuck it, no need this, im not a contributor.

---------

Signed-off-by: sleepyyapril <123355664+sleepyyapril@users.noreply.github.com>
Co-authored-by: sleepyyapril <123355664+sleepyyapril@users.noreply.github.com>
(cherry picked from commit 1ef0bd0f47aadfb3b0ea8f48c8c693c3f9118ad4)

* [Shitmed] Increase Brain Eating Time & Show Brain Eating Popup (#1563)

# Description

Increases the delay when eating brains from 1 second to 5 seconds, and
when someone starts or finishes eating a brain it shows a red popup to
everyone in range.

Eating a character's brain round-removes them. However, it's easy to do
with a 1-second timer and very easy to accidentally RR someone through
eating their brain by fat-fingering the "Activate item in hand" keybind.
(Disclaimer: I did exactly this LOL). This PR will make accidental brain
eating very unlikely.

A red popup on eating is also important so everyone in range will be
aware if someone might get RR'd through brain eating and be able to try
to intervene.

This also fixes a _very hilarious_ bug where the target of a force-feed
wouldn't get the popup showing the flavors of the food after the force
feed was complete.

## Media

**Popup on self when eating a brain**

https://github.com/user-attachments/assets/66cf0952-4005-4c3f-bfed-585829f78369

**Popup on others eating a brain**

https://github.com/user-attachments/assets/bd183de2-c39f-49bd-a633-52cba7b961b1

**Popup on others force-feeding a brain**

https://github.com/user-attachments/assets/459042de-42ae-4064-9f43-be2d7f0c73d0

**Popup of flavors on being force-fed**

https://github.com/user-attachments/assets/c593263f-96b6-40b0-875c-0dc0b160cd62

## Changelog

:cl: Skubman
- tweak: Eating a brain now takes 5 seconds (from 1 second), and feeding
someone a brain takes 6 seconds.
- add: Attempting or succeeding to eat a brain now shows a red popup
message to everyone else in range, e.g. "Urist McHands is trying to eat
the brain!" / "Urist McHands ate the brain!".
- fix: Fixed a bug where players after being force-fed were not shown a
popup showing the food's flavors.

(cherry picked from commit 536deb6a3a18ec802bae134bec814aaba7f0a243)

* Handheld Frequencies (#1574)

<!--
This is a semi-strict format, you can add/remove sections as needed but
the order/format should be kept the same
Remove these comments before submitting
-->

# Description

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

Adds frequencies to handheld radios. Ports [this
PR](https://github.com/new-frontiers-14/frontier-station-14/pull/1833)
with changes.

![image](https://github.com/user-attachments/assets/e39d8fce-7fa3-4e0a-918a-506b0446de35)

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears
in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- add: Added handheld radio frequencies, accessible by pressing your
"Use Item" key while holding one.

---------

Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>
(cherry picked from commit 19a350fc9ae0d958406b3d6018f71ab8af5447c4)

* ChemMaster+ (#1585)

<!--
This is a semi-strict format, you can add/remove sections as needed but
the order/format should be kept the same
Remove these comments before submitting
-->

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

Changes how the ChemMaster works:
1. Removes the amount buttons and instead uses a textbox that resets
whenever a value is entered or focus ends.
2. Shrinks the ChemMaster again.
3. Adds sorting options, including for quantity, last added, and a sort
option for the (default) alphabetical order.
4. Sorting options save via the ChemMaster itself, not per-user.

Video showcase:
https://discord.com/channels/1218698320155906090/1218698321053356060/1330129166384894046

---

<!--
You can add an author after the `:cl:` to change the name that appears
in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- add: Added sorting options to the ChemMaster.
- add: Added the ability to input custom amounts into the ChemMaster via
a textbox that resets on change.
- tweak: The width of the ChemMaster UI has been lowered.
- remove: Removed quantity buttons from the ChemMaster.

---------

Co-authored-by: VMSolidus <evilexecutive@gmail.com>
(cherry picked from commit 9baa214ac1c1628a24470d9aeeeafa1d7893253b)

* ChemMaster Improvements (#1650)

Hate having to use the same buffer for pills? Want sorting and container
solutions for output? No need to wait _any_ longer! A brand new pill
buffer, a sort feature _for_ the new pill buffer **and** you have your
amount buttons back!

Showcase:
https://ptb.discord.com/channels/1218698320155906090/1218698321053356060/1332224976803074123

:cl:
- add: A new ChemMaster experience has been granted to the people of
Einstein Engines. Includes a pill buffer!

(cherry picked from commit 35c26cf0c907542f1580ab4d026d33e98863fd98)

* Hotfix ChemMaster (#1663)

:cl:
- fix: ChemMaster no longer loses your last added progress because of a
silly goose move.

(cherry picked from commit 9a0c6b2f6cf8113a1a738f39479023a469966d7f)

* Fix Chemmaster Two (#1667)

simplify this shitcode

i borked it last time, sorry

---------

Signed-off-by: sleepyyapril <123355664+sleepyyapril@users.noreply.github.com>
Co-authored-by: VMSolidus <evilexecutive@gmail.com>
(cherry picked from commit af62f691553f2981ed2e64c256adb5a23750ec88)

* ChemMaster Shorten Amounts (#1711)

<!--
This is a semi-strict format, you can add/remove sections as needed but
the order/format should be kept the same
Remove these comments before submitting
-->

# Description

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

Only use 1, 5, 10, 15, 20, 25, 30, 50 as default amounts.
Add the ability to add your own as you please.

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears
in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- tweak: Tweaked the ChemMaster to no longer have 300 buttons.

(cherry picked from commit dc0539733636b3b099539d67699b5658bde7e759)

* Fillbot (#1958)

<!--
This is a semi-strict format, you can add/remove sections as needed but
the order/format should be kept the same
Remove these comments before submitting
-->

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

The humble fillbot! Takes items from the ground and inserts them into a
lathe, silo or disposal unit it's linked to!

---

<!--
A list of everything you have to do before this PR is "complete"
You probably won't have to complete everything before merging but it's
good to leave future references
-->

- [x] Make the robot pick up items
- [x] Make the robot insert items
- [x] Make the robot differentiate between trash inputs
- [ ] Make the robot only linkable to the correct ports

---

<!--
This is default collapsed, readers click to expand it and see all your
media
The PR media section can get very large at times, so this is a good way
to keep it clean
The title is written using HTML tags
The title must be within the <summary> tags or you won't see it
-->

<details><summary><h1>Media</h1></summary>
<p>

https://github.com/user-attachments/assets/f0cd26c0-36fb-43e2-8bdb-c100f033668b

https://github.com/user-attachments/assets/3809b033-01a6-4b4f-96e0-f4730dd2e92d

https://github.com/user-attachments/assets/ccf325f3-ee60-42f8-9e68-75fafe596a7b

</p>
</details>

---

<!--
You can add an author after the `:cl:` to change the name that appears
in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- add: Added Fillbot. Link it to a lathe, material silo, ore processor
or disposal unit with a multitool and watch it gather stuff!

---------

Signed-off-by: Timfa <timfalken@hotmail.com>
Co-authored-by: VMSolidus <evilexecutive@gmail.com>
Co-authored-by: sleepyyapril <123355664+sleepyyapril@users.noreply.github.com>
(cherry picked from commit a71192bdf05a3b7a40323bb16983c25be432bfc4)

* Add Magistrate, Dignitary Department, and Minor FTL Fixes (#1536)

This adds the Magistrate to the role list of Central Command VIPs. This
also splits up the Central Command VIPs into their own department named
"Dignitary" for the purpose of making it clear that Central Command VIPs
are above Command and should be responding to orders from Central
Command.

---

Lots of stuff to do here, mostly just transferring Chief Justice's
loadout to also work for Magistrate (most of it). I'll also list the
basics I've done as well.

- [x] Add Magistrate-only Access (for locker).
- [x] Add Magistrate Job, Job Icon, and PlayTimeTracker
- [x] Add Magistrate clothes (ported from PS13)
- [x] Add Magistrate job spawner.
- [x] Add Magistrate Locker Teleporter and Locker.
- [x] Add Magistrate Contents to Locker.
- [x] Add Magistrate loadout (Chief Justice included).
- [x] Add Magistrate stamp and stamped icon.

- [x] Add Dignitary Department.
- [x] Put Magistrate, Blueshield Officer, and Nanotrasen Representative
in Dignitary.
- [x] Add FTL for Dignitary.
- [x] Put Dignitary above Command.

- [x] Add Magistrate job spawners to maps.
- [x] Add a Centcom Wintercoat to NTR and Magistrate on the Glacier
map.\

- [x] Add winter coats for Magistrate and NanoRep on Glacier.
- [x] Fix the category grouping FTL for NTR and BSO.

---

<details><summary><h3>Lobby/Customization Screenshots</h3></summary>
<p>

Jobs Listing

![image](https://github.com/user-attachments/assets/61ebe868-b411-449c-810d-794536b7c76f)
Category Grouping

![image](https://github.com/user-attachments/assets/98a6707b-9631-4a38-892a-492a8b9461d3)

</p>
</details>
<details><summary><h3>Items/Clothes/Entities</h3></summary>
<p>

Stamps (Stamp shows the same in the spawn menu)

![image](https://github.com/user-attachments/assets/7225ccca-a07e-4ae3-bd1f-c2e806a7b377)
Magistrate Robes

![image](https://github.com/user-attachments/assets/abb39f42-e9bf-4038-94ad-418f0c642ff7)
Spawner & Magistrate Robes

![image](https://github.com/user-attachments/assets/cdd38786-222c-48b8-8aaa-332f65dad265)
Access List

![image](https://github.com/user-attachments/assets/f92cc6b5-3884-4a17-b898-095e7968ce2a)
PDA Crew Manifest

![image](https://github.com/user-attachments/assets/38c5dc83-6fdf-41fe-b304-2f294359a87d)
Lockers with Teleporter

![image](https://github.com/user-attachments/assets/cf1346fc-5b8f-4c52-a7a7-33c0c7bbedb6)
Locker with Contents (from Teleported Locker)

![image](https://github.com/user-attachments/assets/9f18290a-07cc-4e4b-93f2-01276c7611b6)

</p>
</details>
<details><summary><h3>Map Spawners</h3></summary>
<p>

Arena

![image](https://github.com/user-attachments/assets/10ba04c1-9d48-46e7-aa1e-700f22f1d169)
Asterisk

![image](https://github.com/user-attachments/assets/a49434ef-2efe-4ae4-ae1e-c405add68704)
Core

![image](https://github.com/user-attachments/assets/0fe868e4-92a9-4100-803b-66bb130ee68c)
Edge

![image](https://github.com/user-attachments/assets/bbeebbc6-2a82-4707-b874-7ef8216ccb03)
Europa

![image](https://github.com/user-attachments/assets/6be71cca-6599-46bb-8f8c-f67be12fdaf2)
Gaxstation

![image](https://github.com/user-attachments/assets/02dfe9bb-98ff-41d5-bf9d-d87a4a05fe60)
Glacier (Winter Coats for NTR and Magistrate as well.)

![image](https://github.com/user-attachments/assets/1849f239-611d-4370-a58d-2d63b9cba03c)
Hammurabi

![image](https://github.com/user-attachments/assets/f8b544be-b8ab-4dd9-b801-d5482d244bb9)
Hive

![image](https://github.com/user-attachments/assets/09514381-4131-491e-aa56-fff7b27d200a)
Lighthouse

![image](https://github.com/user-attachments/assets/a0ddcddb-9133-40b0-9f77-b6d2a8eb0f21)
Meta

![image](https://github.com/user-attachments/assets/695dec69-24f8-4de0-a237-9d24000d2a35)
Pebble

![image](https://github.com/user-attachments/assets/1f9926c9-f5fc-4a4d-a4a0-1cabb15e5907)
Radstation

![image](https://github.com/user-attachments/assets/b61dda0d-5a4c-4463-bc80-508426039c85)
Saltern

![image](https://github.com/user-attachments/assets/57c3e590-09f4-4862-b1d0-193c1db9488e)
Shoukou

![image](https://github.com/user-attachments/assets/81a9b185-8d7b-4efa-b565-397f44fe5cd8)
Submarine

![image](https://github.com/user-attachments/assets/2e5860c1-8bea-4730-a629-da348056d7c3)
Tortuga

![image](https://github.com/user-attachments/assets/e45b066a-3180-4dce-af27-bc4af8c2bb59)

</p>
</details>
<details><summary><h3>Misc. Minor Fixes</h3></summary>
<p>

Chief Justice Access

![image](https://github.com/user-attachments/assets/10d54485-d0ee-40bb-b048-738e61bf585f)
Category Groupings (BSO AND NTR)

![image](https://github.com/user-attachments/assets/8548dc33-f75f-4051-84b2-e26230f3cc2c)

![image](https://github.com/user-attachments/assets/dd67ad17-0128-4ca6-bf8d-9b4fded117a5)

</p>
</details>

---

:cl:
- add: Added Magistrate role.
- add: Added Dignitary Department.
- add: Added winter coats for NTR and Magistrate on Glacier Spawn.
- tweak: Moved Blueshield Officer and Nanotrasen Representative to
Dignitary Department.
- fix: Fixed the category groups identifier for BSO and NTR in Loadouts.
- fix: Fixed Chief Justice tag in ID Card Computer.

---------

Co-authored-by: sleepyyapril <123355664+sleepyyapril@users.noreply.github.com>
(cherry picked from commit b8a48704c9a03236230cffd4bd931e2729630d2e)

* Revert "Better Progress Bar (#1648)"

This reverts commit 7b1eea704bbf7d8b0566c89547689eb0ca5cbc01.

* Revert "Floofication of 1587"

This reverts commit 2c536d6e821b646c80db285ca6a9976cd4543b21.

* Revert "Split All CVars Into Separate Files (#1587)"

This reverts commit 0be2cd5f7556286afc708d08b362083fad21078d.

* Partial merge of EE 2136

---------

Signed-off-by: sleepyyapril <123355664+sleepyyapril@users.noreply.github.com>
Signed-off-by: Timfa <timfalken@hotmail.com>
Co-authored-by: Skubman <ba.fallaria@gmail.com>
Co-authored-by: VMSolidus <evilexecutive@gmail.com>
Co-authored-by: Simon <63975668+Simyon264@users.noreply.github.com>
Co-authored-by: DocNITE <docnite0530@gmail.com>
Co-authored-by: sleepyyapril <123355664+sleepyyapril@users.noreply.github.com>
Co-authored-by: Whatstone <166147148+whatston3@users.noreply.github.com>
Co-authored-by: Timfa <timfalken@hotmail.com>
Co-authored-by: John Willis <143434770+CerberusWolfie@users.noreply.github.com>

Commit:
Automatic Changelog Update (#1294)

M3739 committed:
Fix Slime Person Tags (#1307)

* Fix SlimePerson tags

* Update comments

Commit:
Automatic Changelog Update (#1307)
